backend/wmg/pipeline/summary_cubes/calculate_markers.py

```diff
@@ -529,8 +380,8 @@ def _post_process_stats(
         effects[:, np.all(np.isnan(effects), axis=0)] = 0
 
     # aggregate
-    effects = np.nanpercentile(effects, percentile * 100, axis=0)
-    pvals = np.array([stats.combine_pvalues(x[np.invert(np.isnan(x))] + 1e-300)[-1] for x in pvals.T])
+    effects = nanpercentile_2d(effects, percentile * 100, 0)
+    pvals = np.sort(pvals, axis=0)[int(np.round(0.05 * pvals.shape[0]))]
```

Comment:
stats.combine_pvalues is sloooow and we don't use pvalues anyway. replacing it with an approximate "5th percentile" p-value

Reply:
add a comment in the code indicating this is effectively a no-op for the application since pvalues aren't being used